README.md

```diff
@@ -141,15 +143,32 @@ And then execute:
 $ bundle install
 ```
 
-Finally, add these lines to your routes.rb:
+### Update routes file
+
+Add these lines to your routes.rb:
 
 ```ruby
+# config/routes.rb in your main Rails app
+
 require 'rails_drivers/routes'
 
 # This can go before or after your application's route definitions
 RailsDrivers::Routes.load_driver_routes
 ```
 
+This will tell your main Rails app to load the `routes.rb` files generated in each of your drivers.
+
+### application.rb
```

Comment:
I want to make sure this is the correct information. We were unable to get our app to properly render a driver's route without updating our `application.rb`. We figured this out by looking at the app used for testing:

https://github.com/degica/rails_drivers/blob/538fa37fba0562b134dc5dc07cc0fd9695ce51cb/spec/dummy/config/application.rb#L19

Reply:
@tmr08c Hmm, we're not sure why this line is required. I've actually removed it here in https://github.com/degica/rails_drivers/pull/21 and the tests are still passing (maybe our test suite is bad).

This dependency should get included with bundler in your config/application.rb via `Bundler.require(:default, Rails.env)`.

If we could reproduce this and verify the behavior I can add this section. Can I ask what version of Rails you are using? I can setup a quick example app to see if routing is working correctly.

Reply:
🤦 

I'm sorry, I've been getting in the habit of adding gems to my Gemfile with `require: false` and foolishly that didn't click for me when manually adding a `require` to `application.rb`. After removing `require: false` from my Gemfile entry, things worked correctly. I have reverted the commit.